src/Bart/Git/Person.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Bart\Git;
+
+/**
+ * Class Person
+ * Represents a Person in Git, i.e. an Author or a Committer
+ * @package Bart\Git
+ */
+class Person
+{
+    /** @var string $name */
+    private $name;
+    /** @var string $email */
+    private $email;
+
+    /**
+     * Person constructor.
+     * @param string $name
+     * @param string $email
+     */
+    public function __construct($name, $email)
+    {
+        if (filter_var($email, FILTER_VALIDATE_EMAIL) === false) {
```

Comment:
Based on the discussion about the purpose of this class with @asheepapart, it was decided that this class should not validate if the email address is valid. This class is meant to be a simple store of the name/email from the git commit message, whether the name/email are in fact valid or not.